boards/cats_rev1Pro/Core/Src/config/globals.c

```diff
@@ -46,6 +46,15 @@ const ICM20601 ICM3 = {
     .gyro_dps = ICM20601_GYRO_RANGE_2000_DPS,
 };
 
+SPI_BUS SPI_ACCEL = {.cs_port = CS_ACC_GPIO_Port, .cs_pin = CS_ACC_Pin, .spi_handle = &hspi1, .cs_type = LOW_ACTIVE};
+
+const H3LIS100DL ACCEL = {
+    .spi = &SPI_ACCEL,
+    .power_mode = H3LIS100DL_PM_NM_ODR,
+    .sample_rate = H3LIS100DL_ODR_100,
+    .filter = H3LIS100DL_HPC_8,
```

Comment:
I think we should put H3LIS100DL_HPC_64 there. The highpass filter is used to remove the accelerometer offset. The offset is something which is very slowly varying so a very low cutoff frequency should be good. Otherwise we might remove frequency parts of the signal, giving us a worse estimate.

Reply:
We can also disable it